styles.css

```diff
@@ -638,7 +660,7 @@ body button:hover {
   .contact {
     padding-bottom: 1.5em;
     background-image: url('/img/contactIzq-background.svg'), url('/img/contact-center-desktop.svg'), url('/img/contact-der-desktop.svg');
-    background-position: bottom left, 20em 15em, bottom right;
+    background-position: bottom left, 20em 28em, bottom right;
     background-repeat: no-repeat;
   }
```

Comment:
- Kindly make sure you use the correct background for your contact page to make your design stick to the Figma design as required. Here I provide the [link]( https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468)  for the Figma design so that you can even copy those background images. your design should look to the image show below:+1: :grinning:  :grinning:  :grinning:
 
## Figma design 

![image](https://user-images.githubusercontent.com/95297251/186790170-df41d493-0986-4376-ad4b-d4505739779e.png)

## your design
![image](https://user-images.githubusercontent.com/95297251/186790101-05453d7a-87aa-4e91-ac7a-97ba99deccd5.png)